### ISSUE TITLE
RetroArch completely broken on Dell latitude E6410

### ISSUE THREAD

**Issue by meepingsnesroms** [libretro/RetroArch#5868]
The regular builds(both 32 and 64 bit) crash on launch with no stack trace.
The msvc2017 builds have no cores.
The msvc2010 64bit build has no cores.
The msvc2010 32bit build has a giant flickering black polygon on the screen with the gl driver, black screen with the direct3d driver.

Direct3d is installed.
RetroArch 1.6.9
Windows 10 fall creators update
Dell Latitude E6410 x86-64
Intel HD Graphics (Core i5)

[Video of flicker](https://github.com/libretro/RetroArch/files/1533093/VID_20171205_133558.zip)

**Comment by lodgerz** [comment]
try to set, in your retroarch.conf:

```
xmb_shadows_enable = "false"
menu_shader_pipeline = "0"
```

**Comment by meepingsnesroms** [comment]
Dosent fix the crash, I had cg toolkit installed but it still dident work so I removed it.

I think it has to do with defective gl drivers.

**Comment by meepingsnesroms** [comment]
I was able use gdb with msys2 but the stack trace is not very informative.

Type "apropos word" to search for commands related to "word"...
Reading symbols from C:/Users/Emily/Downloads/RetroArch/retroarch_debug.exe...done.
(gdb) run
Starting program: C:\Users\Emily\Downloads\RetroArch\retroarch_debug.exe
[New Thread 8444.0x4e4]
[New Thread 8444.0x3e8]
[New Thread 8444.0x2298]
[New Thread 8444.0x19f0]
[New Thread 8444.0x12d4]
[New Thread 8444.0x189c]
[New Thread 8444.0x15b8]

Thread 1 received signal SIGSEGV, Segmentation fault.
0x0000000000000000 in ?? ()
(gdb) backtrace
#0  0x0000000000000000 in ?? ()
Backtrace stopped: previous frame identical to this frame (corrupt stack?)
(gdb)

**Comment by bparker06** [comment]
Try switching the video driver to gdi (it's software-only), you'll probably want to use the rgui menu driver with that as well. If that works fine, then you have a gl problem, most likely either the driver or the hardware. So far we haven't seen any similar reports with the gl driver to my knowledge.

**Comment by meepingsnesroms** [comment]
I tried the gdi driver in the msvc2010 32bit build before and it dident work so I dident try it anywhere else, but it fixes the instant crash issue with the mingw build.
That makes it load.
So its a gl issue.

**Comment by meepingsnesroms** [comment]
It now wont load any cores and if I try to close it with the X button on the window it wont close.
This same laptop worked fine on linux.

**Comment by bparker06** [comment]
If mingw is crashing on startup it's almost always a GL issue.

The X button issue with gdi was a bug that wasn't fixed until after 1.6.9, so the latest nightly is required for that. If you're switching between msvc and mingw or 32/64bit, there's no telling what incompatible core it's trying to load now. I would empty out the cores folder and redownload and try again.

Otherwise, what doesn't work about the gdi driver specifically?

**Comment by meepingsnesroms** [comment]
The cores folder was empty when I started, I was using a config from the msvc version that started and had set the core download url to the msvc version.

I downloaded the latest nightly and redist and made a config with only these lines in it.
config_save_on_exit = "true"
video_driver = "gdi"
menu_driver = "rgui"

It now can load a non3d core but it still doesnt show download progress in yellow for core downloads so I need to make a issue for that, otherwise its fine now.
